ITR/data/base_providers.py

```diff
@@ -567,40 +640,66 @@ def _standardize(self, intensities: pd.DataFrame) -> pd.DataFrame:
             return standardized_intensities.T
 
     def _winsorize(self, historic_intensities: pd.DataFrame) -> pd.DataFrame:
```

Comment:
this function is changed so much, that it's hard for me to track, if the logic remains or not.
there might be a regression, but we will not notice it, because we don't cover this function with UT.
may i suggest covering this func with UT?
this way we will know, that we didn't break it

Reply:
Do you want to take a crack at writing some unit tests, or should I?